supporter-product-data/cloudformation/create-table-stack.sh

```diff
@@ -0,0 +1,8 @@
+# aws s3 rm s3://supporter-product-data-export-prod/ --recursive --profile membership
```

Comment:
This script is a quick way to recreate the Dynamo tables used by this project. That is the quickest way to drop all the data if needed.